e2e/src/QRScanner.spec.js

```diff
@@ -1,7 +1,9 @@
 import { reloadReactNative } from './utils/retries'
 import { quickOnboarding, waitForElementId } from './utils/utils'
 
-describe('Given QR Scanner', () => {
+// Re-enable when react-native-reanimated is updated to v3
+// https://linear.app/valora/issue/ENG-76/[wallet]-update-react-native-reanimated-to-v3
+describe.skip('Given QR Scanner', () => {
```

Comment:
if it's an android only issue, I think we can do something like https://github.com/valora-inc/wallet/pull/3932/files#diff-b322e5ef789575e6edde98300f6e8afc87c6db925863e81bcd22d4d30e599f55R8 to disable this only for Android